### ISSUE TITLE
Toogle Key doesn't work

### ISSUE THREAD

**Issue by matidio** [lusakasa/saka-key#154]
Firefox 56, Windows 7
Seems that the toggle key doesn't work. Also changing the key binding doesn't make it work (neither double key nor single key). Except that I didn't understand correctly the term "Toggle Saka".
I try to disable temporarily Saka Key on pages where it conflicts with the pages own key bindings.

**Comment by tim-teufel** [comment]
There is another extension written by the author of Saka Key. Its name is [Saka](https://addons.mozilla.org/en-US/firefox/addon/saka/). That key binding toggles _Saka_, not Saka Key.

**Comment by RanTrycl** [comment]
In Saka Key, the extension shows that "l" will toggle Saka Key off and on, but this doesn't appear to work. There was discussion of this in 2017 and it was thought that future releases would enable it, but I can't find anything since. Anyone know the status?

**Comment by tim-teufel** [comment]
> In Saka Key, the extension shows that "l" will toggle Saka Key off and on, but this doesn't appear to work. 

No, as I wrote above, the "l" key toggles _Saka_, not Saka Key. Saka is (was?) another extension by the same developer. Do you have it installed along with Saka Key?

> There was discussion of this in 2017 and it was thought that future releases would enable it, but I can't find anything since.

Can you provide a link to the discussion?

**Comment by tim-teufel** [comment]
Guys, let me explain in other words:
The "l" key _does not_ toggle or temporarily disable Saka Key. It just invokes another extension for tab management with a similar name. I personally don't use it.
